lib/AstOfLlbc.ml

```diff
@@ -1713,6 +1713,13 @@ let expression_of_rvalue (env : env) (p : C.rvalue) : K.expr =
   | UnaryOp (Cast (CastRawPtr (_from, to_)), e) ->
       let dst = typ_of_ty env to_ in
       K.with_type dst (K.ECast (expression_of_operand env e, dst))
+    (* from FnDef to FnPtr *)
+  | UnaryOp (Cast (CastFnPtr (TFnDef _, TFnPtr _to)), e) ->
```

Comment:
cosmetic: comments referring to a particular branch are located immediately underneath the pattern arm elsewhere in the codebase (so, move the comment one line below)